courses/templates/courses/introduction-to-data-mining.html

```diff
@@ -0,0 +1,148 @@
+{% extends "courses/base.html" %}
+{% load staticfiles %}
+
+{% block title %}
+Introduction to Data Mining
+{% endblock title %}
+
+{% block opengraph-social-image %}
+<meta property="og:type" content="article"/>
+<meta property="og:image" content="https://orange.biolab.si/static/courses/new-header.jpg"/>
+<meta property="og:image:width" content="750"/>
+<meta property="og:image:height" content="303"/>
+<meta property="og:title" content="Workshop - Introduction to Data mining"/>
+<meta property="og:description"
+      content=""/>
+<meta property="article:publisher" content="https://www.facebook.com/orangedm/"/>
+<meta property="article:author" content="https://www.facebook.com/orangedm/"/>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+{% endblock opengraph-social-image %}
+
+{% block text %}
+<img style="width:100%" src="{% static 'courses/dm-course-baylor2.jpg' %}" alt="hero shot"/>
+<h1>Introduction to Data Mining</h1>
+<p>Data science is an interdisciplinary field that deals with scientific methods, processes,
+    and systems to extract knowledge or insights from data. The course will introduce machine learning and
+    data visualization techniques to uncover hidden patterns in data. </p>
+
+<h2>Why should I attend?</h2>
+<ul style="margin-bottom: 2em;">
+    <li>You will learn essential approaches in data science that include data exploration, visualization,
+        unsupervised machine learning, classification and regression.
+    </li>
+    <li>After completing the course, you will be able to analyze your own data and use them to develop predictive
+        models.
+    </li>
+    <li>The course will be hands-on and will introduce concepts and methods from data science through analysis of
+        real-life data from biomedicine.
```

Comment:
For better style, I'd suggest:
_The course will be hands-on and will introduce data science concepts and methods through the analysis of real-life data from biomedicine._
Even biomedical data, but this would be pushing it. ;)